scripts/Dockerfile.pbbuilder

```diff
@@ -2,8 +2,5 @@ FROM golang:1.15
 
 ENV GO111MODULE=on
 
-RUN apt update; apt install -y protobuf-compiler && \
-    go get google.golang.org/grpc/cmd/protoc-gen-go-grpc@v1.0.1 && \
-    go get google.golang.org/grpc@v1.32.0 && \
-    go get google.golang.org/protobuf/cmd/protoc-gen-go@v1.25.0 && \
-    go get golang.org/x/tools/cmd/goimports
\ No newline at end of file
+COPY scripts/protoc.sh /protoc.sh
+RUN /protoc.sh deps
```

Comment:
I'm not a huge fan of pulling these out of the docker file. Perhaps this is something we should have a broader discussion on.

Reply:
sure. If the concern is that they aren't in the docker image, they are. just being called from a script. These deps are installed in multiple places so a single source of truth for versions was the motivation for this change.